packages/turf-tin/bench.js

```diff
@@ -1,17 +1,12 @@
-global.tin = require('./');
-var Benchmark = require('benchmark');
-var fs = require('fs');
+const Benchmark = require('benchmark');
+const fs = require('fs');
+const tin = require('./');
 
-global.points = JSON.parse(fs.readFileSync(__dirname+'/test/Points.geojson'));
+const points = JSON.parse(fs.readFileSync(__dirname+'/test/Points.json'));
```

Comment:
When refactoring, use the `path.join()` method to make it friendly for cross platform (Windows).

```javascript
const path = require('path');
const points = JSON.parse(fs.readFileSync(path.join(__dirname, 'test', 'Points.json')));
```